client/verta/verta/dataset/_path.py

```diff
@@ -45,17 +48,39 @@ class Path(_dataset._Dataset):
         ])
 
     """
-    def __init__(self, paths, enable_mdb_versioning=False):
+    def __init__(self, paths, base_path=None, enable_mdb_versioning=False):
```

Comment:
Be careful about breaking API (e.g. order of arguments). No big deal in this case

Reply:
ack